public/walkthroughs/walkthrough-2/walkthrough.adoc

```diff
@@ -15,6 +15,7 @@
 :3Scale-ProductVersion: 2.3
 :EnMasse-master-ProductLongName: EnMasse
 :walkthrough: Integrating API-driven applications
+:fuse-flights-aggregator-app-name: fuse-flights-aggregator
```

Comment:
@aidenkeating i logged all the available attributes but there wasn't one for the aggregator app name. Should there be one?

Reply:
@pb82 There wouldn't be one for the app name as that's walkthrough specific. There should be one for the route though

Reply:
Yes, `route-fuse-flights-aggregator-host` works for the route